README.md

```diff
@@ -270,10 +270,12 @@ $ kubectl get pod example-etcd-cluster-0000 -o yaml | grep "image:" | uniq
 Check the other two pods and you should see the same result.
 
 ## Backup an etcd cluster
```

Comment:
Can you combine these two sections into one section?

```
## Backup and restore etcd cluster

> etcd backup/restore operators are provided as example implementations for how to do ...

....

```

Reply:
I rephrase the above to `The provided etcd backup/restore operators are example implementations`.